### ISSUE TITLE
Hierophant Club and Bag of Holding, Punishment for Gamers Only.

### ISSUE THREAD

**Issue by mixzombie** [tgstation/tgstation#62700]
I would like it if I wasn't Punished for being able to defeat megafuana AND be able to help on the station during the same shift. Hours after defeating the Heirophant, someone begs over comms to come and do toxins, after helping out and getting us both a bag of holding, We get on the shuttle and i use the club. Little did I know that Someone updated it, Deciding that Anyone who has both is crossing the line and has to robust? How dare someone have benefits thru hard work and strife . I get "bluespaced" to a random location on the Z level and miss the shuttle.

TLDR;  It would be nice if I wasn't punished for doing nice things. And fuck icebox also.

**Comment by SuperNovaa41** [comment]
Can you edit this to remove all your crying? no one cares

**Comment by mixzombie** [comment]
I cannot, crying about icebox basically is what keeps me going. As for bringing up yet another limitation put on the hierophant club, and asking that it be changed back is completely reasonable. You acting as a bully to shame me isn't exactly nice. Thanks Seth

**Comment by SuperNovaa41** [comment]
You could've just edited it lol
